include/oneapi/dpl/pstl/tuple_impl.h

```diff
@@ -456,15 +612,41 @@ struct tuple<>
 
     tuple operator[](tuple) { return {}; }
     tuple operator[](const tuple&) const { return {}; }
+    operator tuple_type() const { return tuple_type{}; }
     tuple&
     operator=(const tuple&) = default;
     tuple&
     operator=(const ::std::tuple<>& /*other*/)
     {
         return *this;
     }
-    friend bool
-    operator==(const tuple& /*__lhs*/, const tuple& /*__rhs*/)
+    friend constexpr bool
+    operator==(const tuple& __lhs, const tuple& __rhs)
```

Comment:
Unused function parameters should better be unnamed, to avoid possible compilation warnings.

Reply:
removed, thanks